policy-test/src/curl.rs

```diff
@@ -85,6 +95,42 @@ impl Runner {
         }
     }
 
+    /// Runs a [`k8s::Pod`] with a `curl` container which can execute HTTP
+    /// requests using `kubectl exec`.
+    ///
+    /// The pod:
+    /// - has the `linkerd.io/inject` annotation set, based on the
+    ///   `linkerd_inject` parameter;
+    /// - does not execute any requests unless [`Execable::get`] or
+    ///   [`Execable::exec`] are called on the returned [`Execable`] handle.
+    pub async fn run_execable(&self, name: &str, inject: LinkerdInject) -> Execable {
+        let pod = k8s::Pod {
+            metadata: Self::curl_meta(&self.namespace, name, inject),
+            spec: Some(k8s::PodSpec {
+                service_account: Some("curl".to_string()),
+                containers: vec![k8s::api::core::v1::Container {
+                    name: "curl".to_string(),
+                    image: Some(Self::CURL_IMAGE.to_string()),
+                    command: Some(vec!["sh".to_string(), "-c".to_string()]),
+                    args: Some(vec!["while true; do sleep 60; done".to_string()]),
```

Comment:
super nitpicky but I wonder if we could just replace this with
```suggestion
                    command: Some(vec!["sleep".to_string(), "infinite".to_string()]),
```

(`infinite` seems to work for me with this image, `infinity` doesn't for some reason)

Reply:
oh, nice!